indico/htdocs/js/indico/Core/Dialogs/Users.js

```diff
@@ -121,8 +121,12 @@ type ("FoundPeopleList", ["SelectableListWidget"], {
             var userEmail = Html.span({id: self.id + "_" + pair.key + "_email", className: "email"}, Util.truncate(peopleData.get("email"), 40));
             var affiliation = Html.$($('<span class="affiliation">').text(peopleData.get('affiliation')));
 
-            if (this.showToggleFavouriteButtons && IndicoGlobalVars.isUserAuthenticated && peopleData.get('_type') == "Avatar") {
+            if (this.showToggleFavouriteButtons && IndicoGlobalVars.isUserAuthenticated
+                                                && (peopleData.get('_type') == "Avatar" || peopleData.get('_type') == 'ContributionParticipation')) {
                 var favouritizeButtonDiv = Html.div("actions", new Html(create_favorite_button(peopleData.get('id')).get(0)));
+                if (peopleData.get('_type') == 'ContributionParticipation') {
```

Comment:
I'm not sure I understand the sense of this `if`. Are you sure you don't want to let the condition fall into the `else` loop of the enclosing `if`?